packages/core/src/utils.ts

```diff
@@ -1,6 +1,7 @@
 import ajv from "ajv"
 
 import { Doctype } from "@ceramicnetwork/common"
+import {TileDoctype} from "@ceramicnetwork/doctype-tile";
```

Comment:
maybe we can just import the type here?

Reply:
what do you mean?  You mean using `import type { TileDoctype } from ..."?  I've never been clear on what `import type` actually does

Reply:
Okay just did some reading.  I guess `import type` means the import only really happens at compile time to do type checking, but it doesn't actually import the code behind the type into the generated javascript at the end?

Reply:
Yep exactly!